users/tests.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+def test_user_create():
```

Comment:
you need to think about edge cases and add more tests
what happens if you try to add the same user twice?
what happens with "weird" usernames?
role that is longer than the designated 10 chars?
etc..

Reply:
Alright tomorrow we will work on it